Dockerfile

```diff
@@ -1220,15 +1290,15 @@ RUN \
     ln -s ../libexec/rust/lib/rustlib ${d}/rustlib ; \
   done
 
+# Instead of a symlink to libexec, we must select by go version.
+COPY ./helpers/go/go.sh /usr/bin/go
+COPY ./helpers/go/gofmt.sh /usr/bin/gofmt
+COPY ./helpers/go/gofips.sh /usr/bin/gofips
```

Comment:
I would say that these are "wrappers" rather than "helpers". Could we move these three to a wrappers directory and drop the `.sh` to help with the distinction?

Reply:
works for me